java/src/main/java/com/nedap/retail/example/Application.java

```diff
@@ -309,7 +314,7 @@ private static void sendBlinkRequest(final BufferedReader inputBuffer)
     }
 
     private static void subscribeToEvents(final BufferedReader inputBuffer)
-            throws MessageParsingException, HttpRequestException {
+            throws InputParsingException, HttpRequestException {
```

Comment:
the method `parseEventTypes` still logs "Unsupported option value" in stead of throwing an `InputParsingException`